src/pybind/mgr/cephadm/module.py

```diff
@@ -1988,6 +1988,7 @@ def matches_network(host):
             get_hosts_func=self._get_hosts,
             get_daemons_func=self.cache.get_daemons_by_service,
             filter_new_host=matches_network if daemon_type == 'mon' else None,
+            active_daemon_func=self.cephadm_services[daemon_type].get_active_daemon,
```

Comment:
Instead of adding this function: Another idea would be to extend `DaemonDescription` to contain an `is_active` property. Two benefits would be:

1. We expose this information in a machine readable format for users
2. We avoid adding another source of information for the scheduler

Reply:
Where do you think would be a good place to set this property? I'm not really sure when it is decided that a certain mgr or mds will be the active one. Would it be enough to just add a new section to the serve loop (maybe while refreshing hosts and daemons) that runs the get_active_daemon function with a list of all mgr and mds daemons respectively and sets the flag on the the active one?

Reply:
Also, I will work on adding a test.